document/core/syntax/instructions.rst

```diff
@@ -601,7 +601,7 @@ The |MEMORYSIZE| instruction returns the current size of a memory.
 The |MEMORYGROW| instruction grows a memory by a given delta and returns the previous size, or :math:`-1` if enough memory cannot be allocated.
 Both instructions operate in units of :ref:`page size <page-size>`.
 The |MEMORYFILL| instruction sets all values in a region of a memory to a given byte.
-The |MEMORYCOPY| instruction copies data from a source memory region to a possibly overlapping destination region in another or the same memory.
+The |MEMORYCOPY| instruction copies data from a source memory region (second *immediate* |memarg|) to a possibly overlapping destination region in another or the same memory (first *immediate* |memarg|).
```

Comment:
Can we follow the style already used for table.copy? Also, note that the immediates are memory indices, whereas memargs are a different thing.

Reply:
Uh, sorry, `|memarg|` indeed does not make sense here. I use the same phrasing as for table.copy now.